File: Programming-Languages-Concepts-and-Paradigms/Laboratory/02-lab/02-exercise/FToC.go
package main

import "fmt"

func main() {
	var (
		fahrenheit float64
		celsius float64
	)
	fmt.Print("Enter the temperature in Fahrenheit: ")
	fmt.Scanf("%f", &fahrenheit)

	celsius = (fahrenheit - 32) * 5/9

	fmt.Printf("%.2f Fahrenheit = %.2f Celsius\n", fahrenheit, celsius)
}

File: Programming-Languages-Concepts-and-Paradigms/Laboratory/02-lab/02-exercise/FToC.cc
#include <iomanip>
#include <iostream>

int main() {
  float fahrenheit, celsius;
  std::cout << "Enter the temperature in Fahrenheit: ";
  std::cin >> fahrenheit;
  celsius = (fahrenheit - 32) * 5 / 9;
  std::cout << std::fixed << std::setprecision(2);
  std::cout << fahrenheit << " Fahrenheit = " << celsius << " Celsius"
            << std::endl;
  return 0;
}